src/app/public/modules/date-pipe/fixtures/date-pipe.module.fixture.ts

```diff
@@ -0,0 +1,29 @@
+import {
```

Comment:
Shouldn't this file be named `date-pipe-fixtures.module.ts`  instead of `...module.fixture`?

Reply:
We added a regular expression in Builder to ignore any file that ends in `*.fixture.ts` when running `skyux build`. This naming convention hopes to fulfill that requirement.